src/store/modules/actions/wallet.js

```diff
@@ -79,3 +81,12 @@ export async function initIfNecessary({ dispatch }) {
     await dispatch('initWallet', { accounts, offlineSigner, method });
   }
 }
+
+export async function walletFetchLIKEBalance({ commit }, address) {
+  const c = await getNFTQueryClient();
+  const client = await c.getQueryClient();
+  const balance = amountToLIKE(
+    await client.bank.balance(address, LIKECOIN_CHAIN_MIN_DENOM)
+  );
```

Comment:
I think this part can be refactored to util
There will be some scenarios for getting the balance of an address